app/presenters/track.rb

```diff
@@ -0,0 +1,39 @@
+module ExercismWeb
+  module Presenters
+    class Track
+
+      def initialize(track_id)
+        @track_id = track_id
+        @track = Trackler.tracks[track_id]
```

Comment:
That's a great starting point, thanks!

I gave some tests a shot [in this commit](https://github.com/exercism/exercism.io/pull/3454/commits/5f210786d618b365eda037f6abf2b860968e677d). The setup feels a little clunky to me, though. What do you think -- any suggestions for cleaning this up?